app/src/androidTest/java/com/android/unio/end2end/UserAccountCreationTest.kt

```diff
@@ -72,20 +70,21 @@ class UserAccountCreationTest : EndToEndTest() {
     assertDisplayComponentInScroll(
         composeTestRule.onNodeWithTag(AccountDetailsTestTags.INTERESTS_BUTTON))
     composeTestRule.onNodeWithTag(AccountDetailsTestTags.INTERESTS_BUTTON).performClick()
+
     assertDisplayComponentInScroll(
-        composeTestRule.onNodeWithTag(InterestsOverlayTestTags.CLICKABLE_ROW + "0"))
+        composeTestRule.onNodeWithTag(InterestsOverlayTestTags.CLICKABLE_ROW + "SPORTS"))
```

Comment:
Thanks for these refactors you've done across multiple files!

Reply:
:+1: